dev/source/docs/making-a-mavlink-wifi-bridge-using-the-raspberry-pi.rst

```diff
@@ -416,22 +416,41 @@ Update hostapd
 --------------
 
 The hostapd version installed by apt-get does not fully support the WiFi
-dongle we are using so we need to update it to a later version. Lets get
-the new version of hostapd by typing the follwing command:
+dongle we are using so we need to update it to a later version. 
+
+To get the new version of hostapd for the Adafruit Dongle type the following command:
```

Comment:
I think this is wrong, it is the contrary, you need to step back to older version that support that chip.